<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotStarterDialog</class>
 <widget class="QDialog" name="RobotStarterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Robot Starter (c) SmallRobotics 2022 </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start-Script:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ScriptEdit">
     <property name="text">
      <string>start_sim.sh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="StartButton">
     <property name="text">
      <string>START</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="KillButton">
     <property name="text">
      <string>KILL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
